### DISCUSSION TITLE
RCI

### DISCUSSION THREAD

**Discussion by sunflowerecnu** [jazznbass/scan#140] [Q&A] [answered]
How to compute RCI(Hageman and Arrindell)?

**Comment by jazznbass** [comment]
You have to provide one case and also the reliability of the measurement that was applied for the dependent variable.
Example:

```
> rci(exampleAB$Johanna, rel = 0.8)
Reliable Change Index

Mean Difference =  19.53333 
Standardized Difference =  1.678301 

Descriptives:
         n     mean       SD       SE
A-Phase  5 54.60000 2.408319 1.077033
B-Phase 15 74.13333 8.943207 3.999524

Reliability =  0.8 

95 % Confidence Intervals:
           Lower    Upper
A-Phase 52.48905 56.71095
B-Phase 66.29441 81.97226

Reliable Change Indices:
                             RCI
Jacobson et al.         18.13624
Christensen and Mendoza 12.82426
Hageman and Arrindell   18.49426
```

Please check the helppage with  `?rci`

Best,
Jürgen

**Reply by sunflowerecnu** [reply]
Thanks a lot. I want to  know the  formula. From Hageman & Arrindell (1993) , I still don't understand it.

**Reply by sunflowerecnu** [reply]
I can compute the two RCIs using the formula.
Jacobson et al.         18.13624
Christensen and Mendoza 12.82426

**Reply by jazznbass** [reply, marked as answer]
Ok.
```
SE.A <- sA * sqrt(1 - rel)
SE.B <- sB * sqrt(1 - rel)

(mB - mA) * rel + (mB - mA) * (1 - rel) / 
           (sqrt(rel) * sqrt(2 * SE.A * SE.A))
```

m is the mean of the respective phase.
s is the SD of the respective phase
SE is the standard measurement error of the respective phase
rel is the reliability.

**Reply by sunflowerecnu** [reply]
I got it. Thanks a lot.

**Comment by jazznbass** [comment]
It has been over 10 years that I implemented the RCI fucntion into scan. 
I reread the original paper of Hageman and came to the conclusion that I would not recommend to use the present implementation of the Hageman index in scan.
Hageman estimates the reliability of the differenc scores and therefore needs a correlation of pre and post scores of a lot of cases. Which you usually don't have in single-case designs. I simply ignored this aspect in scan which was not a good solution.
The original Jacobson or the improvement by Christensen had single-case designs in mind and are more appropriate (and straight forward).
I will implemenet a reworked version of the rci function in one of the next scan updates.
Cheers,
Jürgen

**Reply by sunflowerecnu** [reply]
Cheers.
